release/train_tests/benchmark/image_classification/factory.py

```diff
@@ -88,11 +91,26 @@ def collate_fn(self, batch):
         return batch["image"], batch["label"]
 
 
+class ImageClassificationTorchDataLoaderFactory(TorchDataLoaderFactory):
+    """Factory for creating PyTorch DataLoaders for image classification tasks."""
+
+    def __init__(self, benchmark_config: BenchmarkConfig):
+        train_urls = IMAGENET_PARQUET_SPLIT_S3_DIRS["train"]
+        val_urls = IMAGENET_PARQUET_SPLIT_S3_DIRS["train"]
+        super().__init__(
+            benchmark_config,
+            train_urls,
+            val_urls,
+            limit_validation_rows=benchmark_config.limit_validation_rows,
+        )
```

Comment:
If we do the comment above, then we also don't need to pass the url through the init, but rather override the `get_iterable_datasets` method here.